src/server/dfly_main.cc

```diff
@@ -701,6 +711,46 @@ void PrintBasicUsageInfo() {
   std::cout << endl;
 }
 
+void ParseFlagsFromEnv() {
+  if (getenv("DFLY_PASSWORD")) {
+    LOG(WARNING)
+        << "DFLY_PASSWORD environment variable is being deprecated in favour of DFLY_requirepass";
+  }
+  // Allowed environment variable names that can have
+  // DFLY_ previx, but don't necessirily have an ABSL flag created
+  std::set<std::string_view> allowed_environment_flag_names = {"dev_env", "password"};
```

Comment:
Nit: Not that it matters greatly in this case, but we tend to prefer `absl::flat_hash_set` over `std::set`